library/src/main/java/pyxis/uzuki/live/naraeimagepicker/activity/NaraePickerActivity.kt

```diff
@@ -138,4 +137,19 @@ class NaraePickerActivity : AppCompatActivity() {
         finish()
     }
 
+    override fun onStart() {
+        super.onStart()
+        try {
```

Comment:
Should be `ignored: Throwable` or provide detail message or use `catchAll` inline function.

Personally, I preferred `catchAll` function.

```Kotlin
inline fun catchAll(action: () -> Unit) {
        try {
            action()
        } catch (t: Throwable) {
            Log.e("NaraeImagePicker", "Catch an exception. ${t.message}", t)
        }
    }
```